src/main/java/com/study/realworld/user/presentation/model/UserLoginRequest.java

```diff
@@ -0,0 +1,35 @@
+package com.study.realworld.user.presentation.model;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.study.realworld.user.application.model.UserLoginModel;
+
+import javax.validation.constraints.NotBlank;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.WRAPPER_OBJECT;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME;
+
+/**
+ * @author Jeongjoon Seo
+ */
+@JsonTypeName("user")
+@JsonTypeInfo(include = WRAPPER_OBJECT, use = NAME)
+@Getter
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```
access level을 protected로 걸어도 괜찮을것 같아요 :)

Reply:
👍

Reply:
이거 어제 질문하셨던게 여기에도 포함되는거였네요 :)
여기는 entity와 마찬가지로 jackson 또한 기본생성자 + getter를 기본 스펙으로 제시하기 때문에 기본생성자를 protected로 막는거 자체는 entity와 동일한 이유로 사용되었다고 할 수 있어요 😄